packages/spotlight/src/spotlight.js

```diff
@@ -361,6 +361,12 @@ const Spotlight = (function () {
 		}
 	}
 
+	function handleKeyboardStateChangeEvent ({visibility}) {
```

Comment:
Is `visibility` the state of VKB?

Reply:
Yes. When the state of the VKB changes (shows/hides), we receive a custom `keyboardStateChange` event that contains a `visibility` property that reflects the state of the VKB.